src/NzbDrone.Core/Notifications/Webhook/WebhookPayload.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace NzbDrone.Core.Notifications.Webhook
+{
+    internal class WebhookPayload
+    {
+        public string EventType { get; internal set; }
+        public WebhookSeries Series { get; internal set; }
+        public WebhookEpisodeFile EpisodeFile { get; internal set; }
+        public WebhookEpisode Episode { get; internal set; }
```

Comment:
> couldn't we just have a list of episodes that contains the file info as well

yea, that seems a lot cleaner to me. 

I need to figure out how to get restsharp to exclude nulls, but ongrab won't have any file information which kinda makes sense to me? I think i can massage the data to get the quality and release/scene group